pkg/serverless/aws/arn.go

```diff
@@ -55,17 +81,42 @@ func SetRequestID(reqID string) {
 	defer currentReqID.Unlock()
 
 	currentReqID.value = reqID
+	cacheRequestID(reqID)
```

Comment:
Do we need to save this file every time? Or just during a timeout?

Reply:
I think we can probably get away with just saving the values during timeouts/shutdowns. The only scenario that wouldn't be covered is if the extension process is killed and restarted without warning, which isn't supposed to happen.